pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -96,8 +96,8 @@ func NewKubeGenericRuntimeManager(
 	cpuCFSQuota bool,
 	runtimeService internalApi.RuntimeService,
 	imageService internalApi.ImageManagerService,
-) (*kubeGenericRuntimeManager, error) {
-	kubeRuntimeManager := &kubeGenericRuntimeManager{
+) (*KubeGenericRuntimeManager, error) {
```

Comment:
Let's not export kubeGenericRuntime in this PR and just focus on adding the new flags.